CONTRIBUTING.md

```diff
@@ -132,24 +132,54 @@ In order to build and publish ODK, you need the following:
 
 ## General SOP for ODK release and publication
 
+- There are three types of releases: major, minor and development snapshot.
+  - Major versions include changes to the workflow system.
+  - Minor versions include changes to tools, such as ROBOT our python dependencies.
+  - Development snapshots reflect the current state of the `main` (`master`).
+- They all have slightly different procedures which we will detail below.
+
+### Major releases
+
+Major releases contain changes to the workflow system of the ODK, e.g. changes to the `Makefile` and various supporting scripts (e.g. run.sh, update_repo.sh).
+They require users to update there repository light with `make update_repo`
```

Comment:
```suggestion
They require users to update their repository with `make update_repo`
```

Reply:
Not yet ready for review :D But thanks anyways!